Sources/Build/Error.swift

```diff
@@ -10,4 +10,16 @@
 
 public enum Error: ErrorProtocol {
     case noModules
+    case noProducts
+}
+
+extension Error: CustomStringConvertible {
+    public var description: String {
+        switch self {
+        case .noModules:
+            return "no modules found"
+        case .noProducts:
+            return "no products found (note: modulemaps cannot be build)"
```

Comment:
Can we get a specific error message for the case where it is a `system module package`, and make it customized for that? Note that C language targets can have module maps as well, so it is important to use the term `system module packages` when referring to that specific style.